src/olympia/wsgi.py

```diff
@@ -44,8 +44,9 @@ def application(env, start_response):
 
 # Initialize Newrelic if we configured it
 newrelic_ini = getattr(django.conf.settings, 'NEWRELIC_INI', None)
+newrelic_uses_environment = os.environ.get('NEW_RELIC_LICENSE_KEY', None)
 
-if newrelic_ini:
+if newrelic_ini or newrelic_uses_environment:
```

Comment:
Prepares for using environment variables directly to configure NewRelic, this can be used in CircleCI or Travis later to configure NewRelic and generate reports that are easy to compare.